tests/web/test_assets.py

```diff
@@ -55,18 +55,18 @@ def test_get_assets(client, b):
 
 
 @pytest.mark.genesis
-def test_get_assets_limit(client, b):
```

Comment:
This test should be removed in favour of [this](https://github.com/bigchaindb/bigchaindb/pull/2338/files#diff-5887167ed95ccb195dfe41eb9fd2b03cR119). 

NOTE: `b.me` wasn't replaced [here](https://github.com/bigchaindb/bigchaindb/pull/2338/files#diff-5887167ed95ccb195dfe41eb9fd2b03cR68) and it wasn't flagged in Travis.